packages/manager/modules/trusted-nic/src/label/label.less

```diff
@@ -0,0 +1,12 @@
+ovh-manager-trusted-nic-label {
```

Comment:
Is there a reason that you use `less` here, and `scss` for another module in your PR ?

Reply:
No reason actually. This has been changed to `scss` too :)